rosbag2_performance/rosbag2_performance_benchmarking/src/writer_benchmark.cpp

```diff
@@ -188,7 +193,7 @@ void WriterBenchmark::create_writer()
     rosbag2_storage::TopicMetadata topic;
     topic.name = queue->topic_name();
     // TODO(adamdbrw) - replace with something more general if needed
-    topic.type = "std_msgs::msgs::ByteMultiArray";
+    topic.type = "rosbag2_performance_benchmarking_msgs::msgs::ByteArray";
```

Comment:
@carlossvg Delimiters with double colon doesn't work.
It is expected to have `/`  backslash as delimiter.  
i.e. **"rosbag2_performance_benchmarking_msgs/msg/ByteArray"**

Reply:
done